api/singapore/v1alpha1/clusterregistrar_types.go

```diff
@@ -0,0 +1,53 @@
+// Copyright Red Hat
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ClusterRegistrarSpec defines the desired state of ClusterRegistrar
+type ClusterRegistrarSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
```

Comment:
Assume you have to run more than `make` if this is updated?  Maybe `make generate`?

As mentioned previously, there will most likely be kubeconfig data, possibly context name?, and maybe some other uuid or identifier field.